invenio_communities/communities/services/service.py

```diff
@@ -205,6 +209,15 @@ def update_logo(self, identity, id_, stream, content_length=None, uow=None):
         record = self.record_cls.pid.resolve(id_)
         self.require_permission(identity, "update", record=record)
 
+        logo_size_limit = self.config.community_logo_size_default_quota
+        if "APP_RDM_COMMUNITY_LOGO_QUOTA" in current_app.config:
```

Comment:
The idea here was to guarantee that at least a default value is provided by the service itself, in case `APP_RDM` does not set the config.

I think `in` operator does not throw an exception when the key is not set. Initially I had:

```python
if current_app.config["APP_RDM_COMMUNTY_LOGO_QUOTA"]:
    pass
```

and it was failing during tests with a `KeyError` exception. Using `x in dict` does not throw an exception and instead returns `false`.

I agree that the validation is indeed neded ( `is_digit()` and `>0`), just in case the config is badly set by the user.